rstcheck.py

```diff
@@ -44,6 +44,12 @@
 import docutils.utils
 import docutils.writers
 
+try:
+    from sphinx.roles import *  # NOQA
```

Comment:
Thanks! But what are these star imports for? Could you add a unit test to `test_rstcheck.py` show that they are needed?